src/cmd/bootstrap.ts

```diff
@@ -158,80 +160,101 @@ const bootstrapValues = async (): Promise<void> => {
       '`otomi.adminPassword` has been generated and is stored in the values repository in `env/secrets.settings.yaml`',
     )
   }
+}
 
+const postSops = async (): Promise<void> => {
   if (existsSync(`${env.ENV_DIR}/.sops.yaml`)) {
     // encryption related stuff
     const file = '.gitattributes'
     await copyFile(`${rootDir}/.values/${file}`, `${env.ENV_DIR}/${file}`)
     // just call encrypt and let it sort out what has changed and needs encrypting
     await encrypt()
   }
+}
 
-  if (!hasOtomi) {
-    debug.log('You can now use the otomi CLI')
+const customCA = async (originalValues: Record<string, any>): Promise<void> => {
+  const d = terminal('customCA')
+  const cm = get(originalValues, 'charts.cert-manager', {})
+
+  if (cm?.customRootCA && cm?.customRootCAKey) {
+    d.info('Skipping custom RootCA generation')
+    return
   }
-  debug.log(`Done bootstrapping values`)
+  d.info('Need to generate custom RootCA')
+
+  // Code example from: https://www.npmjs.com/package/node-forge#x509
+  const keys = pki.rsa.generateKeyPair(2048)
+  const cert = pki.createCertificate()
+  cert.setExtensions([
+    {
+      name: 'basicConstraints',
+      cA: true,
+    },
+    {
+      name: 'keyUsage',
+      keyCertSign: true,
+      digitalSignature: true,
+      nonRepudiation: true,
+      keyEncipherment: true,
+      dataEncipherment: true,
+    },
+  ])
+  cert.publicKey = keys.publicKey
+  cert.serialNumber = '01'
+  cert.validity.notBefore = new Date()
+  cert.validity.notAfter = new Date()
+  cert.validity.notAfter.setFullYear(cert.validity.notBefore.getFullYear() + 10)
+  const attrs = [
+    { name: 'commonName', value: 'redkubes.com' },
+    { name: 'countryName', value: 'NL' },
+    { shortName: 'ST', value: 'Utrecht' },
+    { name: 'localityName', value: 'Utrecht' },
+    { name: 'organizationName', value: 'Otomi' },
+    { shortName: 'OU', value: 'Self-Signed' },
```

Comment:
OU==='Organizational Unit'

```suggestion
    { shortName: 'OU', value: 'Development' },
```